examples/resources/spectrocloud_cluster_eks/resource.tf

```diff
@@ -64,10 +64,12 @@ resource "spectrocloud_cluster_eks" "cluster" {
 
   machine_pool {
     name          = "worker-basic"
-    count         = 1
-    instance_type = "t3.large"
+    count         = 4
+    instance_type = "t3.xlarge"
+    capacity_type = "spot"
```

Comment:
we have a separate example for spot type, let's update that.

Reply:
https://github.com/spectrocloud/terraform-provider-spectrocloud/blob/af95194e7b7e366159ef20b16fe9f45cdf347a08/examples/e2e/eks_spot

Reply:
reverted to old example